dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -336,16 +353,30 @@ public MetadataImport(Context c, DSpaceCSV toImport) {
                         item = wsItem.getItem();
 
                         // Add the metadata to the item
+                        List<BulkEditMetadataValue> relationships = new LinkedList<>();
                         for (BulkEditMetadataValue dcv : whatHasChanged.getAdds()) {
-                            itemService.addMetadata(c, item, dcv.getSchema(),
-                                                    dcv.getElement(),
-                                                    dcv.getQualifier(),
-                                                    dcv.getLanguage(),
-                                                    dcv.getValue(),
-                                                    dcv.getAuthority(),
-                                                    dcv.getConfidence());
+                            if (StringUtils.equals(dcv.getSchema(), "relation")) {
```

Comment:
I'm wary of having a magic namespace that has to have special handling everywhere we work with metadata.

Reply:
Alexander and Tim made the a similar recommendation.
At the moment, this schema for importing relations was only planned for the CSV import.

Reply:
closing/resolving this discussion as it's already taking place in a Alexander's comment above. Please add your feedback above.